indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -419,6 +419,9 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_track_reviews(self, track):
```

Comment:
If there won't be any other use of this method, I'll gladly remove it. Are you sure there won't be any other place where we'll need to get the list of abstract reviews for a specific track?